index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <div class="all">
+            <div class="top-container">
+                <div class="header">
+                    <div class="logo"><a href="index.html" class="logo">My Logo</a></div>
+                    <div class="menu">
+                        <nav>
+                            <a href="index.html"><img src="image/Enabled.png" alt="menu"/></a>
+                        </nav>
+                    </div>
+                </div>
```

Comment:
- [ ] Kindly wrap this section inside a `header` tag as it is the appropriate instead of a generic `div`